content/posts/2023/reana-0.9.2.md

```diff
@@ -0,0 +1,87 @@
+
+### ... and more!
+
+There are more small improvements in this release.
+It's now possible to delete all the runs of a given workflow directly from the web interface. Furthermore, the validation of the REANA specification file has been improved in case of invalid YAML files or unexpected keywords.
+
+Please see the detailed [REANA 0.9.2 release notes](https://github.com/reanahub/reana/releases/tag/0.9.2) for the complete list of all user-oriented changes.
+
+## What's new for the administrators?
+
+### Automounting of CVMFS repositories
+
+While users have been able to mount and access CVMFS repositories from their jobs, it was not possible to configure and access additional repositories other than the ones allowed by REANA.
+
+Thanks to CVMFS CSI v2, REANA 0.9.2 allows users to mount any available repository, as long as CVMFS is correctly configured in the Kubernetes cluster. To learn more about this, you can check out the official docs of CVMFS CSI on [adding CVMFS repository configuration](https://github.com/cvmfs-contrib/cvmfs-csi/blob/master/docs/how-to-use.md#adding-cvmfs-repository-configuration).
+
+### ARM Docker images
+
+Given the rise in interest for ARM computers, both for personal use and in servers, REANA now supports being deployed on ARM machines.
+Starting from this release, the REANA's Docker images published on [DockerHub](https://hub.docker.com/u/reanahub) will contain two variants, supporting `linux/amd64` and `linux/arm64` platforms. This change is transparent, and the correct variant will be automatically chosen by Docker depending on the machine it's running on.
+
+### How to upgrade existing REANA 0.9.1 clusters
+
+<!-- taken and adapted from 0.9.0 blog post -->
+
+If you are a REANA cluster administrator and you would like to upgrade from REANA 0.9.1 to REANA 0.9.2, you can proceed as follows.
+
+First of all, this release does not introduce any new Helm values, so you can keep using your previous configuration without any changes needed.
+Assuming that your custom Helm values are stored in `myvalues.yaml`, you can then use the [Helm diff plugin](https://github.com/databus23/helm-diff) to inspect the forthcoming change, and you can then perform the upgrade using the standard Helm commands:
+
+```console
+$ helm repo update
+$ helm diff upgrade reana reanahub/reana --version 0.9.2 --values myvalues.yaml
+$ helm upgrade reana reanahub/reana --version 0.9.2 --values myvalues.yaml
+```
+
+Additionally, this release includes some changes to the database schema, in order to allow for more than nine restarts of user workflows and to improve the performance of common database queries.
+To perform the database schema upgrade, you can follow the dedicated [upgrade documentation](https://docs.reana.io/administration/deployment/upgrading-db/):
```

Comment:
To perform the database schema upgrade, you can launch the following command after the upgrade, as further explained in the dedicated upgrade documentation: